src/Maestro/Maestro.Web/Pages/Account/AccountController.cs

```diff
@@ -137,6 +135,11 @@ private async Task<ApplicationUser> CreateUserAsync(ExternalLoginInfo info)
             }
         }
 
+        public static bool ShouldAddClaimToUser(Claim c)
+        {
+            return c.Type == ClaimTypes.Email || c.Type == "urn:github:name" || c.Type == "urn:github:url" || c.Type == ClaimTypes.Role;
```

Comment:
I'd probably had made it a switch, but this works. :-)

Reply:
I just used the labmda -> named method refactor and left it alone.

Reply:
Makes sense. :-)  Approved anyway. ;-)